Sources/MockoloFramework/Models/ParamModel.swift

```diff
@@ -17,19 +17,26 @@
 import Foundation
 
 final class ParamModel: Model {
-    var name: String
-    var offset: Int64
-    var length: Int64
-    var type: SwiftType
+    internal init(label: String, name: String, type: SwiftType, isGeneric: Bool, inInit: Bool, needsVarDecl: Bool, offset: Int64, length: Int64) {
```

Comment:
[Question]
Why is it explicitly marked internal?

Reply:
It doesn't have any particular meaning. It gets added when I used 'Generate Memberwise Initializer'. I left it as is.